dumpling/export/status.go

```diff
@@ -60,7 +63,16 @@ func (d *Dumper) GetStatus() *DumpStatus {
 	ret.FinishedBytes = ReadGauge(d.metrics.finishedSizeGauge)
 	ret.FinishedRows = ReadGauge(d.metrics.finishedRowsGauge)
 	ret.EstimateTotalRows = ReadCounter(d.metrics.estimateTotalRowsCounter)
-	ret.CurrentSpeedBPS = d.speedRecorder.GetSpeed(int64(ret.FinishedBytes))
+	ret.CurrentSpeedBPS = fmt.Sprintf("%d MiB/s", d.speedRecorder.GetSpeed(int64(ret.FinishedBytes))/1024)
+	if d.progressReady {
+		progress := float64(d.completedChunks.Load()) / float64(d.totalChunks.Load())
+		if progress > 1 {
+			ret.progress = "100%"
+			d.L().Warn("completedChunks is greater than totalChunks", zap.Int64("completedChunks", d.completedChunks.Load()), zap.Int64("totalChunks", d.totalChunks.Load()))
+		} else {
+			ret.progress = fmt.Sprintf("%5.2f%%", progress)
```

Comment:
Is it printed in the log?

Reply:
it is printed in L38

Reply:
I added a failpoint to log it in test, you can see the result in description, and failpoint is in https://github.com/pingcap/tidb/pull/38625/commits/a67f27442e980cab916a63e3976e1a51bc4b7f39